src/libraries/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -398,6 +399,7 @@ public static void ValidateTransformBlock(byte[] inputBuffer, int inputOffset, i
                 ThrowArgumentOutOfRange(ExceptionArgument.inputCount);
         }
 
+        [DoesNotReturn]
```

Comment:
None of these ThrowXx methods return.  Shouldn't the attribute be on all of them?

Reply:
Well yeah, only this was influencing nullability analyzes, should we annotate all when others has no impact?

Reply:
Given they're all grouped like this, I think it makes sense to attribute all of them.   We haven't been exerting effort to ensure that every annotation/attribute internal to a library is 100% correct, just that the public surface area is 100% correct and the internal attribution is good enough to compile successfully.  But it's easy enough here to do the "right thing".